tests/test_utils/api_connexion_utils.py

```diff
@@ -80,15 +81,30 @@ def create_role(app, name, permissions=None):
     return role
 
 
+def set_user_single_role(app, username, role_name):
+    sm = app.appbuilder.sm
+    role = create_role(app, role_name)
+    user = app.appbuilder.sm.find_user(username)
+    if role not in user.roles:
+        user.roles = [role]
+        sm.update_user(user)
```

Comment:
```suggestion
def set_user_single_role(app, username, role_name):
    role = create_role(app, role_name)
    user = app.appbuilder.sm.find_user(username)
    if role not in user.roles:
        user.roles = [role]
        app.appbuilder.sm.update_user(user)
```

(Or reuse `sm` on line 87. Just nitpicking.)